packages/vscode-extension/src/extension.ts

```diff
@@ -136,6 +137,35 @@ export async function activate(context: ExtensionContext) {
     commands.registerCommand("RNIDE.diagnose", diagnoseWorkspaceStructure)
   );
 
+  async function closeAuxiliaryBar(registeredCommandDisposable: Disposable) {
+    registeredCommandDisposable.dispose(); // must dispose to avoid endless loops
```

Comment:
Nit: may be more readable with helper like `interceptCommand(name: string, fn: () -> Disposable): Disposable`.